app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if isinstance(command, str) and command.split(" ")[0] == "cp":
```

Comment:
The input validation checks if the command starts with 'cp', but it should also verify that the command string contains exactly three elements. Consider splitting the command once and checking the length of the resulting list.